tests/test.ts

```diff
@@ -20,6 +21,107 @@ const test_cases: {
   '々': 'cjk-letter',
 };
 
+// BMP
+const cjkRadicalsSupplement = '⻁⺪⻏';
```

Comment:
It'd be better to unify the test cases in terms of their representation, either the original form or the proposed form should be fine.

Reply:
I unified to the proposed form.